app/javascript/react/containers/AlbumShowContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import AlbumShow from '../components/AlbumShow';
+
+class AlbumShowContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id:"",
+      name:"",
+      artist:"",
+      description:"",
+      release_date:"",
+      genre:"",
+      album_art:""
+    }
+  }
+
+  componentDidMount() {
+    fetch(`/api/v1/genres/${this.props.params.genre_id}/albums/${this.props.params.id}`)
+    .then(response => {
+
+      if (response.ok) {
+        return response;
+      } else {
+        let errorMessage = `${response.status} (${response.statusText})`
+        error = new Error(message);
+        throw(error);
+      }
+    })
+    .then(response => response.json())
+    .then(body => {
+      let fetchedAlbum = body.album
+      this.setState({ id: fetchedAlbum.id, name: fetchedAlbum.name, artist: fetchedAlbum.artist, description: fetchedAlbum.description, release_date: fetchedAlbum.release_date, genre: fetchedAlbum.genre, album_art: fetchedAlbum.album_art })
```

Comment:
This is very difficult to read -- use line breaks like so:
```suggestion
      this.setState({ 
           id: fetchedAlbum.id, 
           name: fetchedAlbum.name, 
           ...
       })
```